pandas/tests/extension/test_numpy.py

```diff
@@ -349,26 +344,6 @@ def test_setitem_sequence_broadcasts(self, data, box_in_series):
         # length than the value
         super().test_setitem_sequence_broadcasts(data, box_in_series)
 
-    @skip_nested
-    def test_setitem_loc_scalar_mixed(self, data):
-        # AssertionError
-        super().test_setitem_loc_scalar_mixed(data)
```

Comment:
Where those already working? (as those are generally skipped, not only for ArrayManager)

Reply:
AFAICT the added check in indexing.py fixed this for both AM and BM.

Reply:
do we need a note for this? e.g. is this a user facing bug fix?

Reply:
no and no